app/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/forms/BusinessNameForm.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.forms
+
+import play.api.data.Form
+import play.api.data.Forms.mapping
+import uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.forms.validation.StringConstraints._
+import uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.forms.validation.utils.MappingUtil._
+import uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.models.BusinessNameModel
+
+
+object BusinessNameForm {
+
+  val businessName = "businessName"
+
+
+  def businessNameValidationForm: Form[BusinessNameModel] = Form(
+    mapping(
+      businessName.trim -> oText.toText.verifying(
```

Comment:
```suggestion
      businessName -> oText.toText.verifying(
```
The trim isn't doing anything